purchases/src/main/kotlin/com/revenuecat/purchases/Backend.kt

```diff
@@ -6,20 +6,30 @@
 package com.revenuecat.purchases
 
 import android.net.Uri
+import com.revenuecat.purchases.attributes.SubscriberAttribute
 import org.json.JSONException
 import org.json.JSONObject
 
 private const val UNSUCCESSFUL_HTTP_STATUS_CODE = 300
 private const val HTTP_SERVER_ERROR_CODE = 500
 
+private const val ATTRIBUTES_ERROR_RESPONSE_KEY = "attributes_error_response"
+private const val ATTRIBUTE_ERRORS_KEY = "attribute_errors"
+
+
+/** @suppress */
```

Comment:
are the warnings supressions still needed? what do they supress?

Reply:
it's for suppressing from the Javadocs